addons/mail/models/mail_thread.py

```diff
@@ -2469,9 +2480,12 @@ def message_notify(self, *,
             return self.env['mail.message']
 
         # preliminary value safety check
+        forbidden_names = {'message_id', 'message_type', 'parent_id'}
+        if 'message_type' in kwargs and kwargs['message_type'] == 'auto_comment':
+            forbidden_names.remove('message_type')
         self._raise_for_invalid_parameters(
             set(kwargs.keys()),
-            forbidden_names={'message_id', 'message_type', 'parent_id'}
+            forbidden_names=forbidden_names
         )
```

Comment:
Gonna need a sanity check on this one @tde-banana-odoo 

It's in this PR when it wasn't in the original because I added the option to set that specifically for calendar (who don't use manual notify anymore).

I did it like this to keep a consistent message and check, just bypassing if the parameter is ok

Reply:
@reth-odoo It is actually a bit strange, as message_notify main purpose is to generate 'user_notification' messages that will not be displayed on public chatter, target only recipients, and should be considered as a "one shot message". Here it will be displayed as a classic message while being created through 'message_post'. I suppose the need is to avoid notifying other followers, but the message is still displayed in chatter ?

Reply:
@tde-banana-odoo indeed I didn't think about it but I guess these messages won't be filtered anymore :/

The need is the same as for the templates, i.e. if I reply to "you have been removed from the meeting" with "why have I been removed", the message isn't just left as a note in the chatter.

But I guess maybe this wasn't the right solution in the end if the behavior is going to be so different.
Shall I backpedal the change in calendar on the previous versions for now? To be consistent
Don't think it's a massive deal to display the messages in this particular case but :shrug: